docs/operational_guide/availability_consistency_durability.md

```diff
@@ -0,0 +1,126 @@
+# Tuning Availability, Consistency, and Durability
+
+## Overview
+
+M3DB is designed as a High Availability [HA](https://en.wikipedia.org/wiki/High_availability) system because it doesn't use a consensus protocol like Raft or Paxos to enforce strong consensus and consistency guarantees.
+However, even within the category of HA systems, there is a broad spectrum of consistency and durability guarantees that a database can provide.
+To address as many use cases as possible, M3DB can be tuned to achieve the desired balance between performance, availability, durability, and consistency.
+
+Generally speaking, [the default and example configuration for M3DB](https://github.com/m3db/m3/tree/master/src/dbnode/config) favors performance and availability, as that is well-suited for M3DB's most common metrics and Observability use cases. To instead favor consistency and durability, consider tuning values as described in (TODO).
+Database operators who are using M3DB for workloads that require stricter consistency and durability guarantees should consider tuning the default configuration to better suite their use case.
+The rest of this document describes the various configuration options that are available to M3DB operators to make those tradeoffs.
+
+While reading the rest of this document, we recommend referring to [the default configuration file](https://github.com/m3db/m3/blob/master/src/dbnode/config/m3dbnode-all-config.yml) (which has every possible configuration value set) to see how the described values fit into M3DB's configuration as a whole.
```

Comment:
since both of these paragraphs reference reading the rest of the doc, we might want to combine them. e.g.:

> The rest of this document describes the various configuration options that are available to M3DB operators to make such tradeoffs. While reading it, we recommend referring to [...]